### PR TITLE
Adds the possibility to alternate between S0 and S3 or S0 and S4.

### PR DESCRIPTION
**Pull request by jloehel**
Signed-off-by: Jürgen Löhel <jloehel@suse.com>

### CODE REVIEW

File: stsx [hunk 1]
```diff
@@ -312,14 +316,14 @@ while [ $c -lt $maxnum ] ; do
...
-  if [ "$method" == "s4" ] ; then
-      resume=20
-      method="s3"
-  else
+  if [ "x$alternate" != "xnone" ] ; then
+    temp=$alternate
+    alternate=$method
+    method=$temp
+    resume=20
+    if [ "$method" =~ "^s4k?$" ] ; then
```
Comment (sndirsch) [inline stsx]: I believe this should be 

  if [[ "$method" =~ ^s4 ]]; then

instead. Right?

**Review by sndirsch — COMMENTED** [review]
I found one issue ...

File: stsx (hunk 1 above)
Comment (jloehel) [reply stsx]: This regex matches only for the strings s4 and s4k. But you are right the script checks the methods before the regex and I have only this set of methods ["s4", "s4k", "s3", "s3k", "s0", "s0k"] for alternate. So the script only needs to check if the string starts with "s4". Thank you for the hint. I will change it.

Comment (sndirsch) [reply stsx]: Sure you've tested this?

#!/bin/bash
method=s4k
if [ "$method" =~ "^s4" ]; then
  echo hello
fi

gives me

./test.sh: line 5: [: =~: binary operator expected

Whereas

#!/bin/bash
method=s4k
if [[ "$method" =~ ^s4 ]]; then
  echo hello
fi

gives me

hello

**Review by sndirsch — COMMENTED** [review]
See latest comment ...

**Comment by jloehel** [comment]
You are right again. I fixed it.

**Pull request merged**